charts/kubernetes/templates/applications.yaml

```diff
@@ -203,6 +208,19 @@ spec:
       value: cloud-config
     - name: secret.hostMount
       value: 'false'
+  - version: 2.31.1
```

Comment:
Is there some reason you haven't cleaned up the old version(s)?  They are totally unused now...

Reply:
Yeah, because we've got existing clusters provisioned with these versions.  I can remove these once they've been upgraded.

Reply:
Which is all well and good until the services restart and the bundle is referencing the new version and the upgrade happens regardless... Drew looks like he's about to nuke everything anyway!

Reply:
I was playing it safe but good point, might as well throw this into the mix.